package.json

```diff
@@ -23,15 +23,15 @@
     "@chakra-ui/react": "1.6.5",
     "@emotion/react": "11.4.0",
     "@emotion/styled": "11.3.0",
-    "@next/bundle-analyzer": "11.0.1",
+    "@next/bundle-analyzer": "11.1.0",
     "@octokit/auth-callback": "2.0.1",
     "@octokit/rest": "18.8.0",
     "@sentry/nextjs": "6.10.0",
     "downshift": "6.1.3",
     "focus-visible": "5.2.0",
     "framer-motion": "4.1.17",
     "lodash": "4.17.21",
-    "next": "11.0.1",
+    "next": "11.1.0",
```

Comment:
I've done my own testing manually and I did not see any obvious breakage, or bad stuff.

I can use the comparator as I'd expect, and the standard query string displays properly.

We do have a dark/light toggle in the deployment preview somehow, is that intended?

Reply:
Ah okay, just wanted to check. 😄